sites/public/lib/helpers.tsx

```diff
@@ -133,9 +136,10 @@ export const getImageTagLabelFromListing = (listing: Listing) => {
 export const getListingTags = (
   listingPrograms: ListingProgram[],
   listingFeatures: ListingFeatures,
+  homeType: HomeTypeEnum,
```

Comment:
can you make this as an optional prop since homeType can be undefined/null? `homeType?: HomeTypeEnum`